app/components/isp-jam-auth/template.hbs

```diff
@@ -13,7 +13,7 @@
         </div>
       </div>
       <div class="form-group">
-        <button disabled={{if (or authenticating (not (and password username))) true false}} class="btn btn-primary" onclick={{action 'authenticate'}}>{{t 'global.continueLabel'}}</button>
+        <button disabled={{disabled}} class="btn btn-primary" onclick={{action 'authenticate'}}>{{t 'global.continueLabel'}}</button>
```

Comment:
We generally encourage slightly more descriptive variable names. (like `allowLogin` or `loginEnabled` or `NScanClickTheShinyButtonZOMGSoShiny`*)

For example, if we ever wanted to have other UI in that component that could also be toggled, the word "disabled" could quickly become ambiguous.